pvt/simulators/weighted-pools/weightedPool.ts

```diff
@@ -0,0 +1,99 @@
+import { simulateSwap } from './swaps';
+import { simulateExactTokensInJoin, simulateSingleTokenJoin } from './joins';
+import { simulateExactTokensOutExit, simulateSingleTokenExit } from './exits';
+import WeightedPool from './model';
+import { SwapKind } from '@balancer-labs/balancer-js';
+
+async function main() {
+  const SWAP_AMOUNT = 100;
+  const WITHDRAWAL_PCT = 0.01;
+  const DEPOSIT_PCT = 0.01;
+  const TOTAL_LIQUIDITY = 100000;
+  const SWAP_FEE = 0.005;
+
+  let i;
+
+  // Create a pool with the specified tokens and weights
+  const pool = new WeightedPool(['BAL', 'WETH'], [0.8, 0.2], SWAP_FEE);
+  // Initialize with total liquidity and prices
+  pool.initialize(TOTAL_LIQUIDITY, [12, 2800]);
+
+  const balIndex = pool.indexOf('BAL');
+  const wethIndex = pool.indexOf('WETH');
+
+  // These "simulate" functions were designed for a very specific purpose - testing the behavior over
+  // the full range of swap fee values (1%-99%). They write CSV files in a format that can be imported
+  // to Excel and graphed.
+  //
+  // Of course it is more general: you can create a simulated pool with any number of tokens and starting
+  // liquidity (it will calculate the balances, given the token prices in USD), and perform operations to
+  // inspect the results.
+  //
+  // Running `yarn hardhat run weighted-pools/weightedPool.ts` from the /simulators directory should produce
```

Comment:
```suggestion
  // Running `yarn ts-node weighted-pools/weightedPool.ts` from the /simulators directory should produce
```

We don't need hardhat in this package as we're not running a chain. We can also remove the config file + dependencies.